static/styles/zulip.css

```diff
@@ -427,12 +427,11 @@ label {
 
 /* Bootstrap's focus outline uses -webkit-focus-ring-color which doesn't exist in Firefox */
 a:not(:active):focus,
-button:focus,
 .new-style label.checkbox input[type="checkbox"]:focus ~ span,
 i.fa:focus,
 i.zulip-icon:focus,
 .auto-select:focus {
-    outline: 2px solid hsl(215, 47%, 50%);
+    outline: 2px auto hsl(215, 47%, 50%);
     /* TODO: change solid to auto once the Chromium bug #1105822 is fixed */
```

Comment:
https://crbug.com/1105822 was marked as a duplicate of https://crbug.com/991480 and closed.
https://crbug.com/991480 was marked as won't fix and closed.
I would expect this to mean that we cannot make this change yet or else we'd regress on #15768, however, when I tested this PR locally, I could not reproduce that bug, hence, I believe we can also just delete the comment (and mention why in the commit message).

Reply:
@YashRE42 I have changed auto to solid, but a bit confused about the thing regrading the comment. It would be helpful if you mention what changes are to be made.